org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/VizGUI.java

```diff
@@ -107,7 +107,7 @@
 	private final JButton		projectionButton, openSettingsButton, closeSettingsButton, magicLayout,
```

Comment:
Can we at some point (after this PR) convert all tabs to spaces?

Reply:
Yup, If you're ok then I'll install an automatic formatter that makes all code look the same. Will be a massive commit though. I'd also like to get rid of all warnings.

Reply:
sounds good